README.md

```diff
@@ -434,23 +434,27 @@ Options:
   -e, [--version=VERSION]
               # The pacticipant version. Must be entered after the
                 --pacticipant that it relates to.
-      [--ignore=IGNORE]
-              # The pacticipant name to ignore. Use once for each pacticipant
-                being ignored. A specific version can be ignored by also
-                specifying a --version after the pacticipant name option.
   -l, [--latest=[TAG]]
               # Use the latest pacticipant version. Optionally specify a TAG
                 to use the latest version with the specified tag.
-      [--to-environment=ENVIRONMENT]
-              # The environment into which the pacticipant(s) are to be
-                deployed
       [--branch=BRANCH]
               # The branch of the version for which you want to check the
+                verification results.
+      [--main-branch=MAIN_BRANCH]
```

Comment:
would you just need to say `--main-branch` which should have been set by `create-or-update-pacticipant` rather than them specifying what the main branch is in this query

Reply:
so it would be a bool opt?

Reply:
Oh, that should be --main-branch|--no-main-branch. I'll fix that. Good pick.